src/app/docs/[lang]/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { DocsLayout } from '../components/docs-layout/docs-layout';
+import { fetchMenu } from '../utils/fetchMenu';
+import { Language } from '@/shared/types';
+
+export async function generateMetadata() {
+  return { title: 'RS School Overview' };
+}
+
+export async function generateStaticParams(): Promise<{ lang: Language }[]> {
+  return [
+    { lang: 'en' },
+    { lang: 'ru' },
+  ];
+}
+
+async function fetchDocs(lang: Language) {
+  const indexDoc = 'README';
+
+  let indexContent = '';
+
+  if (indexDoc) {
```

Comment:
am i wrong or this branch is always true?

Reply:
yep, forgot to remove it after refactoring